pursuit.cabal

```diff
@@ -85,7 +85,7 @@ library
                  , directory                     >= 1.1        && < 1.3
                  , warp                          >= 3.0        && < 4
                  , data-default
-                 , aeson                         >= 0.6        && < 0.10
+                 , aeson                         >= 0.6        && < 0.12
```

Comment:
Okay, now I know what you mean. Does it affect the running Travis build when I overwrite this commit / push another one? (seeing as that takes quite a while and we don't really need to run it another time for this change because it already picks aeson 0.11.2.0)